### PR TITLE
RIA-9174 Enabling OOC scenario for submit appeal OOC and out of time

### PR DESCRIPTION
**Pull request by Ryan-Nesbitt**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/RIA-9174

### Change description ###

- Enabling OOC scenario for Submit Appeal Out of Time letter notification
- Updated unit and functional tests

### Checklist

- [X] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by ikirsanov — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/letter/AppellantInternalCaseSubmitAppealOutOfTimeWithExemptionLetterPersonalisation.java [hunk 1]
```diff
@@ -43,8 +43,10 @@ public String getTemplateId() {
 
     @Override
     public Set<String> getRecipientsList(final AsylumCase asylumCase) {
-        return Collections.singleton(getAppellantAddressAsList(asylumCase).stream()
-            .map(item -> item.replaceAll("\\s", "")).collect(Collectors.joining("_")));
+        return inCountryAppeal(asylumCase) ? Collections.singleton(getAppellantAddressAsList(asylumCase).stream()
```
Comment (MiguelMartinRojas) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/letter/AppellantInternalCaseSubmitAppealOutOfTimeWithExemptionLetterPersonalisation.java:46]: I think we are using this same structure for several notifications maybe is time to refator this into a utility method?

**Review by MiguelMartinRojas — COMMENTED** [review]
I left a comment is it up to you to do it.

**Review by MiguelMartinRojas — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/letter/AppellantInternalCaseSubmitAppealOutOfTimeWithExemptionLetterPersonalisation.java (hunk 1 above)
Comment (Ryan-Nesbitt) [reply src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/letter/AppellantInternalCaseSubmitAppealOutOfTimeWithExemptionLetterPersonalisation.java]: Thanks I will add this and replace the personalisations we already have as well

**Review by MiguelMartinRojas — APPROVED** [review]
Now looks better, thanks

**Pull request merged**